fontforge/splineutil2.c

```diff
@@ -2410,6 +2413,10 @@ SplineSet *SplineSetJoin(SplineSet *start,int doall,real fudge,int *changed) {
 return(start);
 }
 
+SplineSet *SplineSetJoin(SplineSet *start,int doall,real fudge,int *changed) {
```

Comment:
There's not that many calls to SplineSetJoin, can we just change all function calls instead of 'overloading' it here?

Reply:
done